mix.exs

```diff
@@ -15,8 +15,8 @@ defmodule PhoenixEcto.Mixfile do
 
      # Docs
      name: "Phoenix/Ecto",
-     docs: [source_ref: "v#{@version}",
-            source_url: "https://github.com/phoenixframework/phoenix_ecto"]]
```

Comment:
@josevalim: I generated docs successfully without them. I do not used these attributes before, but these attributes describes a source of docs and we want to generate docs (not link to outside source), so we do not need external docs source. Please correct me if I'm wrong.